flink-runtime-web/web-dashboard/src/app/share/customize/job-badge/job-badge.component.spec.ts

```diff
@@ -22,10 +22,8 @@ import { JobBadgeComponent } from './job-badge.component';
 
 describe('A job badge', () => {
   it('displays the state', async () => {
-    await render(JobBadgeComponent, {
-      componentProperties: {
-        state: 'RUNNING'
-      }
+    await render(`<flink-job-badge [state]="'RUNNING'"></flink-job-badge>`, {
```

Comment:
Why is this change necessary? From reading the changelog, it seems that only the `template` key was removed, but we're not using that. The examples on the website still use the same syntax as we did before.

Reply:
I am referring this [changelog](https://github.com/testing-library/angular-testing-library/releases/tag/v11.0.0). 

![image](https://user-images.githubusercontent.com/50307523/150294733-366026a7-58f8-4967-b4b2-af76a5e77765.png)

Reply:
But that only says the `template` property has been removed. We didn't use that. I think the way this was written before should still work.

Reply:
Yes, I think you are right. I will revert it.